---
layout: post
title: USARIEM MRE Study
author: Robert Player
---

<hr>

This blog will cover all aspects of JHUAPL's work on the USARIEM MRE Study:

* Study Overview; objectives, hypotheses, design, and sampling
* Study Metadata; PostgreSQL data ingest, and how the extraction plates were laid out for the 480 samples and controls
* Metadata Analysis; discover covariates and correlations
* 16S Analysis; analyses carried out on the sequence data (variation, PCoA of distances, DE)

*all shell code is performed in bash unless specified otherwise.*

<hr>

# STUDY OVERVIEW

All information and diagrams derived from PDF received from USARIEM:

*15-12-HC; Effects of Meal, Ready-to-Eat (MRE) consumption on gut health. PI: J. Philip Karl, Protocol template version 27 Feb 14*

### Objectives:

The primary objective of this study is to determine the impact of consuming MREs as the sole source of subsistence for 21 d on gut barrier integrity.

Secondary objectives are to determine the impact of consuming MREs as the sole source of subsistence for 21 d on gut microbiota composition, inflammation, and nutritional status.

### Hypotheses:

1. Consuming MREs as the sole source of subsistence for 21 days will decrease gut barrier integrity.
2. Decreased gut barrier integrity will occur concomitant to and be associated with reduced gut microbiota diversity and increased inflammation.
3. Consuming MREs as the sole source of subsistence for 21 days will not adversely affect nutritional status.

### Study Design:

Study activities will primarily occur at USARIEM in Natick, MA.

The study design will be a randomized, parallel-arm, controlled trial. 

Participants recruited from the U.S. Army Natick Soldier Systems Center (NSSC) and surrounding community will be randomized to one of two study groups:
 
* The control group will not receive any intervention, but will be instructed to continue following their normal diet and activity patterns throughout the study. 
* The intervention group will be provided with and instructed to consume MREs and nothing else (except for water) for 21 consecutive days, and then eat their normal diet for 10 consecutive days. Individuals in the MRE-group will also be asked to continue following their normal activity patterns throughout the study.

### Participant Population:

Men and women (not pregnant or lactating) 18-62 y and with a BMI ≤ 30 kg/m2 will be recruited from the NSSC Human Research Volunteer (HRV) pool, from the active duty military personnel stationed at NSSC, from members of the NSSC workforce, and members of the surrounding community. Complete data will be collected from a minimum of 60 volunteers. However, up to 80 volunteers will be enrolled to account for attrition. APL received samples from a total of 61 volunteers.

### Control Group:

Volunteers assigned to the control group will participate in all study data collection procedures, but will not receive any food or beverage products during study participation. Instead control volunteers will be asked to continue consuming their usual diet throughout the study. Including a control group that continues their usual diet (as opposed to using subjects as their own controls and/or a controlled diet), though a potential source of additional variability, is intended to:

1. allow causal relationships between MRE consumption and changes in study outcomes to be inferred with greater confidence
2. assess how the collective attributes of the MRE (e.g., nutrient composition, processing, etc.) relative to the normal diet, rather than the isolated effects of these characteristics, influence study outcomes
3. control for temporal variation in study outcomes unrelated to MRE consumption

### MRE Group:

Volunteers assigned to the MRE group will be provided with 2-3 MREs/d to ensure that estimated daily weight maintenance energy needs are met, and will be instructed not to consume any additional foods or beverages other than water. An exception will be allowed for coffee. Volunteers will be allowed to substitute coffee within the MRE with an equivalent volume of black coffee of their choosing. 

Study staff will use a rotating daily menu of no more than 6 separate MREs from which volunteers will choose 2 (or 3) MREs. The rotating menus will be planned so that regardless of the menus selected, the macronutrient and fiber content of the diet will be relatively consistent from day-to-day.

Individual ration components may be supplemented in individual diets if required to prevent weight loss. When necessary, MREs offered to individual volunteers will be modified to accommodate any dietary aversions or restrictions. 

### Sampling:

Blood, urine, and fecal samples will be collected periodically before, during, and after the intervention for assessment of gut microbiota composition, gut barrier integrity, inflammation and nutritional status.

Volunteers will meet with study staff at least 3d/wk to be weighed, review food records and for other study-related activities.

### Test Days:

On the morning of “test” days (study days 0, 10, 21, and 31) volunteers will report to the testing location following an overnight fast, and having abstained from alcohol use and vigorous exercise for 24 hr and smoking, vaping and tobacco use for 12 hr. After a trained technician draws a single blood sample by antecubital venipuncture, volunteers will then be asked to consume a solution of sucrose, mannitol, sucralose and lactulose dissolved in water.

### Fecal Sample Collection and Storage:

Sealable plastic bags, a GasPak or similar product (to create an anaerobic environment), and a cooler with ice packs will be provided for sample collection. Volunteers will defecate into the collection container, add the GasPak, close the container, and place the container in provided sealable plastic bags with ice packs. Samples will be stored and transported in the provided cooler.

Fecal samples will be collected on 2 separate occasions during baseline (the 10 d period prior to study day 0) [timepoint_id: BL1, BL2], weekly during the first 21-d of the study period [timepoint_id: INT1, INT2, INT3], and then twice during the final week of the study [timepoint_id: POST1, POST2], see Study Timeline table below. Volunteers will be given a 2-3 d window for collecting each sample. However, if a volunteer does not provide a “fresh” sample (i.e., within 12 hr of defecation) within that window, the collection period will be extended until a fresh sample of sufficient mass (≥70 g) is produced. If necessary, MRE consumption will also be extended beyond 21d until a fresh sample of sufficient mass is collected. Study schedules will be planned to minimize the possibility of weekend fecal sample collection; however, the sample collection period will extend into weekends if needed to obtain a “fresh” sample. If a volunteer defecates at a time when he/she cannot return the sample to study staff within 12 hr, then that volunteer will be instructed not to collect that sample, and the collection period will be extended until a fresh sample of sufficient mass is provided to study staff.

Samples will be processed as soon as possible and within 12 hr of defecation. Aliquots (< 20 g total/sample) for 16s rRNA gene sequencing, SCFA analysis and archiving will be frozen at -80&deg;C. One of the archived samples will be designated for bacterial RNA and metabolomics analyses. Because of the high costs associated with these analyses, primary results will be analyzed before deciding whether RNA or metabolomics analysis would be informative and feasible. The remaining sample will be coded only with subject identification number (participant_id) and collection time point information, and will be provided to the US Army Natick Soldier Research, Development and Engineering Center (NSRDEC) Warfighter Directorate’s Bioprocessing and Bioengineering Laboratory.

### Study Timeline:

<img width="100%"; src="{{site.url}}/images/blog/USARIEM-MRE_Study_20170601/usariem_mre_studytimeline.PNG ">

### 16S Sequencing:

Sample Prep:

* 16S rRNA V3-V4 amplicons
* Randomized plating with technical replicates in triplicate
* 300x300bp paired-end, 600 cycle Illumina MiSeq run

### Expected 16S Data Analysis:

Quoted directly from pdf:

"Gut microbiota composition
Taxonomic data will be visualized using hierarchical average-linkage clustering and principal coordinates analysis of beta (i.e., between samples) diversity scores (e.g., Bray-Curtis, and weighted and unweighted UniFrac). Alpha (i.e., within-sample) diversity will be calculated by Shannon, Simpson and Chao1 indices. Changes in alpha diversity over time, and the effect of MRE consumption on changes over time will be examined using mixed models ANCOVA and general linear models with correlated errors as described above.

Changes in the relative abundance of individual taxa over time, and the effect of MRE consumption on changes over time will be examined using mixed models ANCOVA. To control for interindividual variability, the average relative abundance of individual taxa measured in baseline fecal samples will be included as a covariate in individual models. The Benjamini-Hochberg correction will be used to control the false discovery rate for main effects of time, group and the time-by-group interactions resulting from taxa-specific models. When a statistically significant time-by-group interaction is detected (Q < 0.05), post hoc within- and between-group comparisons will be completed and the familywise error rate adjusted using the Bonferroni correction.

Mixed models ANOVA will be used to examine longitudinal associations between individual taxa and biological outcomes. These unadjusted models will include subject as a random factor, and diet, time and individual taxa as fixed factors. Likelihood ratio tests will be used to compare unadjusted models to models adjusted for age, sex, BMI and physical activity. False discovery rate will be controlled using the Benjamini-Hochberg correction.

Data analysis will be completed using SPSS, R, Qiime (28) or similar software as needed."









# STUDY METADATA (Data Structures and Upload)

USARIEM MRE metadata was obtained through a spreadsheet titled: `USARIEM_MRE_postgres_upload.xlsx`.

Working directory on odin@:

```
cd
/data/home/playera1/Projects/USARIEM-mre_study
```

### Table 1

[apl_usariem_mre_volunteers]

This table contains 'volunteer_data' for each participant who volunteered for the study. There are a total of 61 volunteers from which APL received samples from.


| Column Name | Datatype | Description | Units |
|:--- |:--- |:--- |:--- |
| participant_id | SMALLINT | USARIEM ID for volunteer | |
| dropped_from_study | BOOLEAN | USARIEM indicated the volunteer to be dropped from study | |
| mre_group | usariem_mre_group (see below) | 'MRE' or 'control' | |
| sex | text | 'male' or 'female' | |
| age | SMALLINT | age in years | years |
| bmi | FLOAT | Body Mass Index | kg/m^2 |
| weight_day0 | FLOAT | weight taken on first day of study (day 0) | kg |
| weight_day10 | FLOAT | weight taken on day 10 | kg |
| weight_day21 | FLOAT | weight taken on day 21 | kg |
| weight_day31 | FLOAT | weight taken on last day of study (day 31) | kg |

To add to PostgreSQL, create enumerated type for 'mre_group':

```shell
pgdb -c "CREATE TYPE usariem_mre_group AS ENUM ('control','MRE');"
```

Create the schema for this table (set PK and which columns cannot have a NULL field):

```shell
pgdb -c "CREATE TABLE apl_usariem_mre_volunteers (
	participant_id SMALLINT NOT NULL,
	dropped_from_study BOOLEAN,
	mre_group usariem_mre_group,
	sex text,
	age SMALLINT,
	bmi FLOAT,
	weight_day0 FLOAT,
	weight_day10 FLOAT,
	weight_day21 FLOAT,
	weight_day31 FLOAT,
	PRIMARY KEY (participant_id)
);"
```

Create tsv file:

```
touch data_apl_usariem_mre_volunteers.tsv
```

then copy/paste data from excel (be sure to remove headers):

Ingest tsv into created table:

```
pgdb -c "\copy apl_usariem_mre_volunteers (participant_id, dropped_from_study,	mre_group, sex, age, bmi, weight_day0, weight_day10, weight_day21, weight_day31) FROM '$PWD/data_apl_usariem_mre_volunteers.tsv' DELIMITER E'\t' CSV NULL AS '';"
```



### Table 2

[apl_usariem_mre_samples]

This table contains data for each fecal sample collected at 7 different time points per participant (some timepoints are missing a collection). There were 435 anticipated fecal collections, 420 of which were made, 419 were received by APL (missing participant_id '29', timepoint '2'), and participant_id '20' is indicated as 'dropped_from_study' however APL will be sequencing the 5 samples received (missing POST1 and 2) having this participant_id. This table also contains DNA extraction plate location information (plate, row, and column), as well as sample IDs for NAIC, REIC, and additional PCR control samples. The total number of records in this table is 480, which cover 5 x 96 well plates.


| Column Name | Datatype | Description | Units |
|:--- |:--- |:--- |:--- |
| apl_sid | text | full APL Sample ID | |
| apl_sid_trunc | SMALLINT | truncated, 4 digit APL Sample ID | |
| plate | SMALLINT | Plate number used during DNA extraction | |
| plate_row | text | Row letter of plate | |
| plate_column | SMALLINT | Column number of plate | |
| box_id | text | USARIEM Box ID | |
| box_location | SMALLINT | USARIEM Box Location | |
| cryotube_id | text | USARIEM Cryotube ID | |
| usariem_sample_id | text | USARIEM Sample ID |
| participant_id | SMALLINT | USARIEM Participant ID | |
| timepoint | SMALLINT | Time point fecal sample collected | |
| timepoint_id | usariem_mre_timepoint_id | ID designation for timepoint (see Study Overview)| |
| sample_received_by_apl | BOOLEAN | Did APL receive the sample? | |
| aliquot_mass | SMALLINT | mass of sample in cryotube | mg |
| time_between_defecation_and_freezing | SMALLINT | yep | minutes |

To adding to PostgreSQL, create enumerated type:

```shell
pgdb -c "CREATE TYPE usariem_mre_timepoint_id AS ENUM ('BL1','BL2','INT1','INT2','INT3','POST1','POST2');"
```

Create the schema for this table (set PK, FK [to link with apl_usariem_mre_volunteers table] and which columns cannot have a NULL field):

```shell
pgdb -c "CREATE TABLE apl_usariem_mre_samples (
	apl_sid text NOT NULL,
	apl_sid_trunc SMALLINT NOT NULL,
	plate SMALLINT,
	plate_row text,
	plate_column SMALLINT,
	box_id text,
	box_location SMALLINT,
	cryotube_id text,
	usariem_sample_id text NOT NULL,
	participant_id SMALLINT,
	timepoint SMALLINT,
	timepoint_id usariem_mre_timepoint_id,
	sample_received_by_apl BOOLEAN,
	aliquot_mass SMALLINT,
	time_between_defecation_and_freezing SMALLINT,
	PRIMARY KEY (apl_sid),
	FOREIGN KEY (participant_id) REFERENCES apl_usariem_mre_volunteers(participant_id)
);"
```

Create tsv file:

```
touch data_apl_usariem_mre_samples.tsv
```

then copy/paste data from excel (be sure to remove headers):

Ingest tsv into created table:

```
pgdb -c "\copy apl_usariem_mre_samples (apl_sid, apl_sid_trunc, plate, plate_row, plate_column, box_id, box_location, cryotube_id, usariem_sample_id, participant_id, timepoint, timepoint_id, sample_received_by_apl, aliquot_mass, time_between_defecation_and_freezing) FROM '$PWD/data_apl_usariem_mre_samples.tsv' DELIMITER E'\t' CSV NULL AS '';"
```


# METADATA ANALYSIS

covariates and correlations



# 16S ANALYSIS

analyses carried out on the sequence data (variation, alpha/beta diversity, PCoA of distances, DE)

















